app/javascript/components/App.js

```diff
@@ -1,15 +1,36 @@
 import React from "react"
 import PropTypes from "prop-types"
-import { BrowserRouter as  Router, Route, Switch, NavLink } from "react-router-dom"
-
-import { Nav, NavItem } from "reactstrap"
+// import { BrowserRouter as  Router, Route, Switch, NavLink } from "react-router-dom"
+//
+// import { Nav, NavItem } from "reactstrap"
 // import "application.scss"
 
```

Comment:
You will not need to import the scss file.

Reply:
okay